src/components/space-facilities/space-facilities.vue

```diff
@@ -18,6 +18,19 @@
         {{ $t(getFacilityValue(facility)) }}
       </span>
     </li>
+    <li v-if="seats" class="space-facility__item space-facility__seating">
+      <svg-icon
+        v-tooltip="{
+          content: seatsDescription,
+          trigger: 'hover click focus'
+        }"
+        name="seat-icon"
+        class="space-facility__seating-icon"
+      />
+      <h4 class="a11y-sr-only">
```

Comment:
You should switch the `<h4>` and the icon, as the info belonging to this heading should always come after a heading.